src/test/java/de/uhd/ifi/se/decision/management/jira/view/matrix/TestMatrix.java

```diff
@@ -0,0 +1,53 @@
+package de.uhd.ifi.se.decision.management.jira.view.matrix;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.impl.DecisionKnowledgeElementImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestMatrix extends TestSetUp {
+	private Matrix matrix;
+
+	@Before
+	public void setUp() {
+		init();
+		List<DecisionKnowledgeElement> decisions = new ArrayList<>();
+
+		DecisionKnowledgeElement element = new DecisionKnowledgeElementImpl();
```

Comment:
Currently, we have test data that we reuse (in the testdata package) and sometimes test data is created like here. I think we should also create a global graph test data.